pkg/types/installconfig.go

```diff
@@ -29,8 +29,8 @@ type InstallConfig struct {
 	// ClusterID is the ID of the cluster.
 	ClusterID string `json:"clusterID"`
 
-	// Admin is the configuration for the admin user.
-	Admin Admin `json:"admin"`
+	// SSHKey is the public ssh key for the launcher.
```

Comment:
The comment is not correct. This SSHKey is used for providing access to instances..

Reply:
yup, will clarify that